apps/app/src/features/growi-plugin/server/models/vo/github-url.ts

```diff
@@ -24,10 +26,15 @@ export class GitHubUrl {
   }
 
   get archiveUrl(): string {
-    const ghUrl = new URL(`/${this.organizationName}/${this.reposName}/archive/refs/heads/${this.branchName}.zip`, 'https://github.com');
+    const encodedBranchName = encodeURIComponent(this.branchName);
```

Comment:
It needs to encode `#` and `%`, but URL class does not encode these. ref: https://nodejs.org/docs/latest-v18.x/api/url.html#percent-encoding-in-urls